Firestore/core/src/firebase/firestore/remote/exponential_backoff.cc

```diff
@@ -32,6 +32,17 @@ using firebase::firestore::util::AsyncQueue;
 using firebase::firestore::util::TimerId;
 namespace chr = std::chrono;
 
+/**
```

Comment:
All of these constants should go in an anonymous namespace. Once you've added an anonymous namespace to the file, put the using declarations in there too. See `memory_remote_document_cache.mm` for an example of what that looks like.